gen/nvml/vgpu.go

```diff
@@ -334,6 +334,26 @@ func (VgpuInstance VgpuInstance) GetGpuInstanceId() (int, Return) {
 	return VgpuInstanceGetGpuInstanceId(VgpuInstance)
 }
 
+// nvml.VgpuInstanceGetGpuPciId()
+func VgpuInstanceGetGpuPciId(VgpuInstance VgpuInstance) (string, Return) {
+	var Length uint32 = 1 // Will be reduced upon returning
+	for {
+		VgpuPciId := make([]byte, Length)
+		ret := nvmlVgpuInstanceGetGpuPciId(VgpuInstance, &VgpuPciId[0], &Length)
+		if ret == SUCCESS {
+			return string(VgpuPciId[:clen(VgpuPciId)]), ret
```

Comment:
Question, why use `clen` here instead of the returned value of `Length`? Or is that only set if it is too short? (the comments seem to indicate that this is the case)

Reply:
I think it is only set if it is too short (maybe). I know I ran into issues with other similar functions which is why I introduced the custom `clen` function to begin with. This follows the pattern of all other such functions.